xray/test/test_plotting.py

```diff
@@ -38,6 +39,11 @@ def contourf_called(self, plotmethod):
         paths = plt.gca().findobj(mpl.collections.PathCollection)
         return len(paths) > 0
 
+    def pcolormesh_called(self, plotmethod):
```

Comment:
Looking back through the code I'm not sure we need these methods. Maybe simpler just to do `self.assertTrue(isinstance(artist, mpl.collections.<name>))`